src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/DependenciesGraphProvider.cs

```diff
@@ -342,11 +331,39 @@ private static GraphNodeId GetTopLevelGraphNodeId(string projectPath, string mod
                 GraphNodeId.GetPartial(CodeGraphNodeIdName.File, filePath));
         }
 
-        private static readonly ConcurrentDictionary<(int id, Guid guid), string> s_iconNameCache = new ConcurrentDictionary<(int id, Guid guid), string>();
-
-        private static string GetIconStringName(ImageMoniker icon)
+        private sealed class GraphIconCache
         {
-            return s_iconNameCache.GetOrAdd((icon.Id, icon.Guid), i => $"{i.guid:D};{i.id}");
+            /// <summary>The set of icons that have already been registered.</summary>
```

Comment:
Do you find these comments useful? I don't - they are pretty much just restating the name of the field.

Reply:
No they're not useful. IIRC they moved with the members.